packages/react-notion-x/src/components/asset.tsx

```diff
@@ -269,17 +269,18 @@ export const Asset: React.FC<{
         alt={alt}
         zoomable={zoomable}
         height={style.height as number}
+        width={style.width as number}
```

Comment:
@paultalarian doesn't width require units (like px)?

Reply:
From what I see, the width and height proeprties are strings (the initial value ff width is '100%', which is string). Are we sure we are not breaking something here by forcing wrong type?

Reply:
Good question, height was already defined as is, so I just followed the same logic. 
From my tests, everything works fine, but I agree it looks odd.